docs/getting_started/tutorial01.rst

```diff
@@ -13,13 +13,14 @@ Logging in
 Navigate to http://localhost:8000/admin/ and log in with the username
 and password you created with the ``python manage.py createsuperuser`` command.
 
-After logging in, you will be taken to the admin dashboard, also known as the "Wagtail Admin".
+After logging in, you will be taken to the admin dashboard - also known as the "Wagtail Admin".
```

Comment:
Fun fact about sphinx... as a writer you might appreciate this. It transforms a lot of normal characters into fancy punctuation, such as:
* single dash as normal single dash
* double dash `--` into an en-dash
* triple dash `---` into an em-dash.
* triple dots `...` into ellipses character.

No change needed, but for example if you're looking for an em-dash here you could use `---`.

Reply:
Cool